atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/sweep/metrics/TargetedSweepMetricsTest.java

```diff
@@ -271,43 +341,55 @@ public static void assertSweepTimestampConservativeEquals(long value) {
         assertThat(getGaugeConservative(AtlasDbMetricNames.SWEEP_TS).getValue()).isEqualTo(value);
     }
 
-    public static void assertLastSweptConservativeEquals(long value) {
-        assertThat(getGaugeConservative(AtlasDbMetricNames.SWEPT_TS).getValue()).isEqualTo(value);
+    public static void assertLastSweptTimestampConservativeEquals(long value) {
+        assertThat(getGaugeConservative(AtlasDbMetricNames.LAST_SWEPT_TS).getValue()).isEqualTo(value);
     }
 
-    public static Gauge<Long> getGaugeConservative(String name) {
+    private void assertMillisSinceLastSweptConservativeEquals(long lastSwept) {
+        assertThat(getGaugeConservative(AtlasDbMetricNames.LAG_MILLIS).getValue()).isEqualTo(clockTime - lastSwept);
+    }
+
+    public static void assertMillisSinceLastSweptConservativeLessThanOneSecond() {
+        assertThat(getGaugeConservative(AtlasDbMetricNames.LAG_MILLIS).getValue()).isBetween(0L, 1000L);
+    }
+
+    private static Gauge<Long> getGaugeConservative(String name) {
         return getGauge(AtlasDbMetricNames.TAG_CONSERVATIVE, name);
     }
 
     public static void assertEnqueuedWritesThoroughEquals(long value) {
         assertThat(getGaugeThorough(AtlasDbMetricNames.ENQUEUED_WRITES).getValue()).isEqualTo(value);
     }
 
-    public static void assertEntriesReadThoroughEquals(long value) {
+    private static void assertEntriesReadThoroughEquals(long value) {
         assertThat(getGaugeThorough(AtlasDbMetricNames.ENTRIES_READ).getValue()).isEqualTo(value);
     }
 
-    public static void assertTombstonesPutThoroughEquals(long value) {
+    private static void assertTombstonesPutThoroughEquals(long value) {
         assertThat(getGaugeThorough(AtlasDbMetricNames.TOMBSTONES_PUT).getValue()).isEqualTo(value);
     }
 
-    public static void assertAbortedWritesDeletedThoroughEquals(long value) {
+    private static void assertAbortedWritesDeletedThoroughEquals(long value) {
         assertThat(getGaugeThorough(AtlasDbMetricNames.ABORTED_WRITES_DELETED).getValue()).isEqualTo(value);
     }
 
     public static void assertSweepTimestampThoroughEquals(long value) {
         assertThat(getGaugeThorough(AtlasDbMetricNames.SWEEP_TS).getValue()).isEqualTo(value);
     }
 
-    public static void assertLastSweptThoroughEquals(long value) {
-        assertThat(getGaugeThorough(AtlasDbMetricNames.SWEPT_TS).getValue()).isEqualTo(value);
+    private static void assertLastSweptTimestampThoroughEquals(long value) {
+        assertThat(getGaugeThorough(AtlasDbMetricNames.LAST_SWEPT_TS).getValue()).isEqualTo(value);
+    }
+
+    private void assertMillisSinceLastSweptThoroughEquals(long lastSwept) {
```

Comment:
This is a confusing name, since we're actually asserting that `MillisSinceLastSweptThoroughEqualsOneHundredMinus(lastSwept)`. Might be better to just pass `clockTime - lastSwept` into the method each time.